subprojects/docs/src/docs/release/notes.md

```diff
@@ -217,12 +217,12 @@ See the user guide section on the [`plugins {}` block](userguide/plugins.html#se
 
 ### Experimental caching for C/C++ compilation
 
-This release adds support for using the build cache with C/C++ compilation.
-To enable that feature the system property `org.gradle.native.caching` has to be set to `true`.
-Caching C/C++ compilation is still experimental and some caveats apply:
+This release adds _experimental_ support for using the build cache with C/C++ compilation.
+To enable this feature the system property `org.gradle.native.caching` has to be set to `true`.
+There are some missing task inputs to the compilation tasks which can cause incorrect cache hits:
```

Comment:
This is a lot better already. I think we should indicate that the current state of affairs is only temporary. Say, "Some inputs to the compilation tasks are not tracked yet, which can cause false cache hits:"